tests/compatibility-test.py

```diff
@@ -29,7 +29,7 @@
 )
 
 # Default Ray version
-ray_version = '2.8.0'
+ray_version = '2.9.0'
```

Comment:
I found that `@unittest.skipIf` uses `ray_version` with the `2.9.0`. We may remove `ray_version` here, and only initialize the global variable in `__main__`.

Reply:
The interesting thing is that when I print `ray_version` in `setUpClass`, the value is 2.8.0.